### PR TITLE
add audio per line, audio padding, progress bars and fix bugs

### PR DESCRIPTION
**Pull request by sawhney17**
This PR fixes bugs with the audio player and redoes parts of the audio component. 

Handled quite a few bugs and cases. 

Also adds new UI to entires without transcriptions.

### CODE REVIEW

File: electron/src/features/entries/EntryEditor.tsx [hunk 1]
```diff
@@ -284,9 +345,22 @@ function EntryEditor() {
...
       content = (
-        <Card shadow="md" p="lg">
-          <Text>No Transcription Found</Text>
-        </Card>
+        <Center>
+          <Stack>
+            <Title order={3}>No Transcription Found</Title>
+            <Button
+              onClick={() => {
+                //@ts-expect-error entry is always going to be defined here
```
Comment (oenu) [inline electron/src/features/entries/EntryEditor.tsx:353]: This should be handled with a check for entry rather than removing the typing

Comment (sawhney17) [reply electron/src/features/entries/EntryEditor.tsx:353]: Oversight on my part. Thought I'd already changed that 😅

Just pushed a fix

Comment (sawhney17) [reply electron/src/features/entries/EntryEditor.tsx]: Added an else clause as well

**Comment by oenu** [comment]
Im testing the audio playback and I think we need to add a little bit more time to the start and end of clips, probably an extra second on either side (excluding the first and last clip)

Plus we might want to automatically remove any clips that have a negative duration

**Comment by sawhney17** [comment]
What do you mean by negative duration?

**Comment by oenu** [comment]
The system used to output negative length clips, it seems to not do that anymore which is good

**Comment by petersterne** [comment]
Wow, the line-by-line audio is very cool! But I don't love how separated all the lines are now. It makes it tougher to quickly scan the transcript and follow the conversation. Would it be possible to just put small play buttons to the left of each line but otherwise the lines right on top of each other, more like how they are in the VTT file?

Also, the main play/pause/log buttons seem stuck near the top of the transcript. Ideally, they should float and follow the user as the user scrolls down. Otherwise, the user has to scroll all the way back up to the top of the transcript in order to pause the overall audio.

**Comment by petersterne** [comment]
Also need to make it so that you can play the full audio once you've imported the file, even if you haven't transcribed it yet.

**Comment by oenu** [comment]
I forsee an issue which might arise when loading an audio track, playing some of it, changing audio padding in settings, and then going back to that track. Something to track down in the future

**Review by oenu — APPROVED** [review]
Looks good

**Comment by sawhney17** [comment]
@petersterne Simply clicking the play button should work. Will add some ui/ux to that.

**Pull request merged**